perf/change_point_detector.go

```diff
@@ -66,6 +66,12 @@ func (spc *signalProcessingClient) DetectChanges(ctx context.Context, series []f
 				Name:    point.Algorithm.Name,
 				Version: point.Algorithm.Version,
 			},
+			CalculatedOn: time.Now(),
+			Measurement: measurement,
+			Triage: model.TriageInfo{
+				TriagedOn: time.Time{},
```

Comment:
this isn't needed, I would rely on default initialization.

Reply:
Default initialization would wipe the "untriaged" status, no?